src/mmda/predictors/heuristic_predictors/svm_word_predictor.py

```diff
@@ -0,0 +1,390 @@
+"""
```

Comment:
oh great point; i put it there since that's where the old word predictor was, but technically this should be something like an `sklearn predictor`. 

i can migrate to its own directory